### ISSUE TITLE
Allow user to share their score on Twitter.

### ISSUE THREAD

**Issue by itaditya** [itaditya/trick-or-treat-game#18]
Next to the user score, show a "Share on Twitter" button. Pressing on it lets user submit their score on Twitter as text.

Additonally- Generate a cool image like a scorecard for user to post on Twitter. This carries more WOW factor.

**Comment by 2pai** [comment]
can we use react-twitter-embed ?

**Comment by itaditya** [comment]
We just need an anchor tag like this https://developer.twitter.com/en/docs/twitter-for-websites/tweet-button/overview

Using a library for it would be an overkill. But thanks for letting me know about this awesome library.

**Comment by bennetthanna** [comment]
I would like to work on this one if that's okay

**Comment by itaditya** [comment]
Just this code is enough for opening Twitter page in new tab with prefilled content

```html
<a href="https://twitter.com/intent/tweet?url=https://trick-or-treat.netlify.com&text=I finished the game in 48 moves" target="_blank" title="Share on Twitter">
  Share on Twitter
</a>
```

No need of any JS. The criteria for acceptance is that the button also looks just like the Share Tweet button. So you have to work on the CSS. The button text should be "Share your score on Twitter"

Just replace the number 48 with the moves the player gets in the game.

**Comment by itaditya** [comment]
fixed by #45
